src/pipecat/services/ai_services.py

```diff
@@ -149,6 +156,10 @@ async def run_tts(self, text: str) -> AsyncGenerator[Frame, None]:
     async def say(self, text: str):
         await self.process_frame(TextFrame(text=text), FrameDirection.DOWNSTREAM)
 
+    async def handle_interruption(self, frame: StartInterruptionFrame, direction: FrameDirection):
```

Comment:
nit: s/handle_interruption/_handle_interruption/

Reply:
nerver mind, i see we override it.

Reply:
done

Reply:
done